packages/storybook-components/src/WithDocsCustom.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withDocs } from 'storybook-readme';
+
+// Example taken from https://github.com/tuchk4/storybook-readme/blob/master/packages/example-react/stories/index.js
+const PreviewComponent = ({ children }) => (
```

Comment:
Can possibly be simplified to:

```const PreviewComponent = props => (
  <div
    style={{
      textAlign: 'left',
      padding: '25px',
      margin: '25px 0',
      boxShadow: '0 0 40px rgba(0, 0, 0, 0.1)',
    }}
    {...props}
  />

Reply:
Thanks for this example! Will keep as explicit child for the moment as you mentioned in person.